packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -4122,6 +4132,7 @@ void main() {
       'TextInput.setClient',
       'TextInput.show',
       'TextInput.setEditableSizeAndTransform',
+      'TextInput.requestAutofill',
```

Comment:
Is it possible to test this at a high level, like by pumping a whole AutofillGroup setup and tapping a text field so that it autofills?

Reply:
I don't know if we can write an integration test for this on Android/iOS yet. @nturgut mentioned it's possible to do so on web.